sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_log_device_command_events_v1/query.sql

```diff
@@ -40,5 +40,4 @@ WHERE
   jsonPayload.type LIKE 'device.command.%'
   -- Device command metrics were first deployed and stable on 2020-07-08;
   -- there is some data for earlier dates but it's from a failed deployment so we don't count it.
-  AND DATE(`timestamp`) >= "2020-07-08"
```

Comment:
I wouldn't change this part either.

Reply:
that was removed by an accident. Thanks 🙇